guide/ch_advanced_use.tex

```diff
@@ -511,7 +511,6 @@ \chapter{Command Line Options}
 -\/-safe-mode or -s  & {[}none{]} & Synonymous to -\/-mesa-mode.\footnotemark[1]\\
 -\/-single-buffer    & {[}none{]} & Use single-buffering. This reportedly avoids screen blanking problems with some Intel GPUs. \\
 -\/-opengl-compat or -C & {[}none{]} & Request OpenGL 3.3 Compatibility Profile. Might fix graphics problems in some driver configurations.\\
```

Comment:
oops I was not clear enough. Please add `\bottomrule` to the end of line 513 now.